src/Maui/Features/Combat/Manage.razor

```diff
@@ -8,7 +8,8 @@
     </MudItem>
     <MudItem xs="12">
         <MudButton Variant="Variant.Outlined" Color="Color.Secondary" OnClick="OnBeginCombatClicked">Begin Combat</MudButton>
-        <MudButton Variant="Variant.Outlined" Color="Color.Tertiary" OnClick="OnEndTurnClicked" Disabled=@(!_combatEncounter.IsCombatActive())>End Turn</MudButton>
+        <MudButton Variant="Variant.Outlined" Color="Color.Tertiary" OnClick="OnPreviousTurnClicked" Disabled=@(!_combatEncounter.IsCombatActive())>Previous Turn</MudButton>
```

Comment:
Now that there’s two, maybe a button group with three buttons might look nice? Turn could be a text variant? Doesn’t have to be a button grp either.

< Prev | Turn | Next >

Thoughts?

Reply:
See PR #70